packages/system/src/Item/Components/ServiceItemSearchInput/ServiceItemSearchInput.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
```

Comment:
A little bit different to the stock input. Specifically no available stock. I also removed the code... maybe that's a mistake but it looked nicer 😁